src/load-stream-over-network.bench.ts

```diff
@@ -0,0 +1,33 @@
+import { createCeramic } from "./create-ceramic.js";
+import { TileDocument } from "@ceramicnetwork/stream-tile";
+import { StreamID } from "@ceramicnetwork/streamid";
+import { CeramicApi } from "@ceramicnetwork/common";
+import { scenario } from "./benchie/benchmark.js";
+import { create } from "domain";
```

Comment:
@stephhuynh18 is it used somewhere? I do not see.

Reply:
Removed